deploy/gke/redaction/README.md

```diff
@@ -68,4 +68,105 @@ When you are done, you can clean up everything you've done with the following st
+
+***NOTE:** The path in `include` points to where the file would be mounted within the kubernetes cluster environment and is therefore different from where it is present on your local machine. More information on this in [the next section](#making-the-json-file-available-within-the-cluster).*
+
+Next, add this receiver in the traces pipeline, so your trace pipeline looks like - 
+```
+ traces:
+      receivers: [otlp, otlpjsonfile]
+      processors: [memory_limiter, batch, resourcedetection/gke, redaction/credit_cards]
+      exporters: [googlecloud, logging]
+```
+ 
+### Making the JSON file available within the cluster 
+
+Simply adding the JSON file within the directory does not grant the running cluster the access to this file. In order to get access to this file within the kubernetes environment, we would be mounting this file as a kubernetes `configMap` for a new deployment - 
+
+1. Make sure that there are no current deployments by running `kubectl get deployments -n $OTEL_NAMESPACE`. If there are active deployments, delete them using `kubectl delete`. 
+2. Since we also updated our collector configuration, we will need to recreate the `otel-config` configMap too. Delete the old configMaps, if any before proceeding. 
+    - To delete configMaps, you can use `kubectl delete configmaps <configMap name> -n $OTEL_NAMESPACE`. 
+3. Recreate `otel-config` configMap with the updated configuration - `kubectl create configmap otel-config --from-file=./otel-config.yaml -n $OTEL_NAMESPACE`. 
+4. Create a new configMap `otlp-test-data` - this would be used to mount our test data in the kubernetes cluster as a volume - `kubectl create configmap otlp-test-data --from-file=./otlp-data/testdata.json -n $OTEL_NAMESPACE`. 
+5. Update the [manifest.yaml](./manifest.yaml) to add `volume` and `volumeMount` configuration for the newly created configMap - `otlp-test-data`. 
+   - Update the `spec.volumes` section to add a new `configMap` entry  
+        ```
+        - configMap:
+                name: otlp-test-data
+            name: test-data-vol   # This could be changed to anything
+        ```
+   - Update the `spec.containers.volumeMounts` section to configure a new volumeMount for the added volume. 
+        ```
+            - name: test-data-vol  # This name should match to the name of volume in spec.volumes
+                mountPath: /mnt/testdata   # This is the location in the kubernetes environment where the file for configMap will be mounted
+        ```
+6. Deploy the cluster using the new deployment manifest - `kubectl apply -f manifest.yaml -n $OTEL_NAMESPACE`.
+7. Verify if the file is available within the cluster at the mount path specified. This can be done by getting into the running pods within the cluster.
+    - `kubectl exec -n $OTEL_NAMESPACE -it <pod_container_name> -- cat /mnt/testdata/testdata.json` - should show the contents of the file
+    - You can get the pod name using `kubectl get pods -n $OTEL_NAMESPACE `
```

Comment:
Ah I see now why the busybox steps were included ... we aren't really encouraged to use busybox due to its license, so we shouldn't include these steps by default. Our goal is to ship a very minimal docker image and busybox adds a lot of weight to that image too.

Please remove the busybox build steps from the dockerfile. These steps can be moved to their own troubleshooting section (a user shouldn't need to verify the file was mounted if everything is working). That troubleshooting section can show how to modify the Dockerfile to include a shell

Reply:
I see, makes sense. I can add those steps in troubleshooting